nyc_data/templates/account/password_reset.html

```diff
@@ -0,0 +1,19 @@
+{% extends 'base.html' %}
+
+{% block titlebar %}
+<h2>Reset Password</h2>
+{% endblock %}
+
+{% block content %}
+
+    <div class="lockdown">
+        <form method="POST" action="{% url 'account_reset_password' %}" class="password_reset">
+            <p>Forgotten your password? Enter your e-mail address below, and we'll send you an e-mail allowing you to reset it.</p>
```

Comment:
Since we won't actually email people, should this just say email `feedback@` or `jiman@cto`?

Reply:
Eh, removed the link to reset passwords, so this is a dead-end template that wouldn't likely be rendered. For future use, I suppose.